developer/src/kmc/src/util/getLastGitCommitDate.ts

```diff
@@ -10,21 +10,53 @@ export const expectedGitDateFormat = /^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(\.\d\d\
  */
 export function getLastGitCommitDate(path: string): string {
   // TZ=UTC0 git log -1 --no-merges --date=format:%Y-%m-%dT%H:%M:%SZ --format=%ad
```

Comment:
Is the commented format no longer applicable?

Reply:
```suggestion
  // git log -1 --no-merges --format=%at
```